app/views/donate/index.html.erb

```diff
@@ -116,17 +116,17 @@ mark,
         </div>
       </div>
       <div class="col-xs-12 pull-md-5 col-md-7">
-        <p class="lead">Journalism starts with the facts: what happened and can you prove it. Getting to the truth can be hard, and convincing others even harder.</p>
-        <p>Investigative journalists use DocumentCloud to sort through piles of primary source documents, find the story, and then publish the evidence for the public. It's been used in some of the most important reporting of the decade, like St. Louis Public Radio's <%= external_link_to 'Ferguson Project', 'http://apps.stlpublicradio.org/ferguson-project/evidence.html' %> and the Pulitzer-winning <%= external_link_to 'An Unbelievable Story of Rape', 'https://www.propublica.org/article/false-rape-accusations-an-unbelievable-story' %>.</p>
-        <p>More than <strong>one million documents</strong> comprised of more than <strong>20 million pages</strong> have been released to the public through DocumentCloud.</p>
+        <p class="lead">Journalism starts with the facts: what happened and what can be proven. Getting to the truth can be hard, and convincing others even harder.</p>
```

Comment:
@knowtheory I get avoiding the casual "can you prove it", but "what can be proven" feels a little passive and awkward.